src/core/js/scriptLoader.js

```diff
@@ -34,24 +45,27 @@
 
     // 3. start loading JQuery, wait for it to be loaded
     function loadJQuery() {
-        Modernizr.load([
-            {
-                load: 'libraries/jquery.v2.min.js',
-                complete: checkJQueryStatus
-            }
-        ]);
+        loadScript('libraries/jquery.v2.min.js', checkJQueryStatus);
     }
 
     //4. Wait until JQuery gets loaded completely then load foundation libraries
     function checkJQueryStatus() {
         if(window.jQuery === undefined) {
             setTimeout(checkJQueryStatus, 100);
         } else {
-            loadFoundationLibraries();
+            setupModernizr();
         }
     }
-    
-    //5. Load foundation libraries and templates then load Adapt itself
+
+    //5. Backward compatibility for Modernizr
+    function setupModernizr() {
+        Modernizr.touch = Modernizr.touchevents;
```

Comment:
Is the declaration of the `touch` property for backward compatibility between v2 and v3?  

In this PR you also refer to `Adapt.device.touch`.  Should this be the preferred property for plugins to check after the bootstrapping?

Reply:
Yes, it's for backward compatibility only. I don't know if there are any other plugins which reference ```Mozernizr.touch```

Reply:
Thanks.  I can't think of a core plugin but I guess there could be non-core ones using it.